play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukCheckboxes.scala.html

```diff
@@ -17,25 +17,35 @@
 @import scala.collection.compat.immutable.LazyList
 
 @import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.snippets._
 
-@this(govukErrorMessage: GovukErrorMessage, govukFieldset: GovukFieldset, govukHint: GovukHint, govukLabel: GovukLabel)
+@this(govukFieldset: GovukFieldset, govukHint: GovukHint, govukLabel: GovukLabel, govukFormGroupSnippet: GovukFormGroupSnippet, govukHintAndErrorMessageSnippet: GovukHintAndErrorMessageSnippet)
 
 @(params: Checkboxes)
 @import params._
 @require(name.nonEmpty, "parameter 'name' should not be empty")
 @require(values.isEmpty || !items.exists(_.checked) || values == items.filter(_.checked).map(_.value).toSet, "checked item(s) conflict with passed value(s)")
-@defining(innerHtmlSnippet(describedBy.getNonEmptyOrElse(""))) { case (describedBy1, innerHtml) =>
-<div class="@toClasses("govuk-form-group", errorMessage.fold("")(_ => "govuk-form-group--error"), formGroupClasses)">
-  @fieldset.fold(innerHtml) { fieldset =>
-    @govukFieldset(Fieldset(
-      describedBy = describedBy1.toOption,
-      legend = fieldset.legend,
-      classes = fieldset.classes,
-      attributes = fieldset.attributes,
-      html = innerHtml
-    ))
+
+@defining(fieldset.fold(describedBy)(fieldset => fieldset.describedBy.filter(_.nonEmpty).orElse(describedBy))) { describedBy =>
```

Comment:
Naming here is a bit confusing. It took me a couple of reads to work out that this is:
`@defining(fieldset.fold(params.describedBy)(fieldset => fieldset.describedBy.filter(_.nonEmpty).orElse(params.describedBy))) { fieldsetDescribedBy =>`

Reply:
Agree this is confusing - unusually this component allows `describedBy` to be provided in 2 ways, which complicates things.  Will rename/qualify things to make it more explicit.